presto-hudi/src/main/java/com/facebook/presto/hudi/HudiPageSourceProvider.java

```diff
@@ -82,38 +83,37 @@ public ConnectorPageSource createPageSource(
     {
         HudiTableLayoutHandle layout = (HudiTableLayoutHandle) layoutHandle;
         HudiSplit hudiSplit = (HudiSplit) split;
-        HudiFile baseFile = hudiSplit.getBaseFile().orElseThrow(() ->
-                new PrestoException(HUDI_CANNOT_OPEN_SPLIT, "Split without base file is invalid"));
-        Path path = new Path(baseFile.getPath());
-
-        Configuration configuration = hdfsEnvironment.getConfiguration(
-                new HdfsContext(session,
-                        layout.getTable().getSchemaName(),
-                        layout.getTable().getTableName(),
-                        baseFile.getPath(),
-                        false),
-                path);
         HudiTableType tableType = layout.getTable().getTableType();
         List<HudiColumnHandle> hudiColumnHandles = columns.stream().map(HudiColumnHandle.class::cast).collect(toList());
         List<HudiColumnHandle> dataColumns = hudiColumnHandles.stream().filter(HudiColumnHandle::isRegularColumn).collect(toList());
 
         final ConnectorPageSource dataColumnPageSource;
         if (tableType == HudiTableType.COW) {
-            dataColumnPageSource = createParquetPageSource(
```

Comment:
After we change HudiSplitManager, we have confident that hudiSplit.getBaseFile().isPresent() is always true. Add an assertion here. We dont need HiveEmptySplitPageSource